Scripts/Mac/PrepFunctionalTests.sh

```diff
@@ -15,34 +15,3 @@ hdiutil attach $GITPUBLISH/*.dmg || exit 1
 GITPKG="$(find /Volumes/Git* -type f -name *.pkg)" || exit 1
 sudo installer -pkg "$GITPKG" -target / || exit 1
 hdiutil detach /Volumes/Git*
-
```

Comment:
Removing this will remove the ability for new devs/machines to install the current Java based GCM. This is fine for now since all of the machines are already configured correctly. Do we expect the new GCM to be coming in so soon that we want to remove this code now?

Reply:
I yanked installation here since I'm not certain that installing the cred manager is the responsibility of this script (named PrepFunctionalTests) now that our functional tests run against the anonymous endpoints.

I also expect the new GCM to be coming in soon for Mac.

If we want scripted installation of the GCM, should we consider creating a script to prep a dev environment? That will keep the functional test prep on ci machines quicker.

Reply:
Oh right, I forgot that we were testing against the anonymous endpoints now. Thanks for the reminder.

In that case, I think it's fine to remove this for now. Later when the new GCM is ready, we should make sure there's an easy way to install it on a dev machine (or maybe we rework our scripts to install Git and VFSForGit from the built installers like we do on Windows).